sparse/impl/KokkosSparse_spadd_symbolic_impl.hpp

```diff
@@ -543,6 +543,7 @@ void spadd_symbolic_impl(
           exec, nrows + 1, c_rowmap_upperbound);
       Kokkos::deep_copy(exec, c_nnz_upperbound,
                         Kokkos::subview(c_rowmap_upperbound, nrows));
+      exec.fence("fence before c_nnz_upperbound used on host");
```

Comment:
```suggestion
          exec, nrows + 1, c_rowmap_upperbound, c_nnz_upperbound);
```

Reply:
Thanks for fixing this, I just suggested to use the version of prefix sum that returns the final element to you (so Kokkos handles the subview, copy and fence)

Reply:
most excellent